pkg/util/signals/signal_posix.go

```diff
@@ -0,0 +1,26 @@
+// +build !windows
```

Comment:
Do we need to support windows? Our code only runs in our docker containers and I don't think we will every use windows.

Reply:
I think we can support it since it is not difficult. And if there are some people want to build the operator in Windows, it should work.